ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -51,6 +51,26 @@ resources:
       cpu: "1"
       memory: 4Gi
 tests:
+- as: alibaba-ipi-fips-p2-f7
+  cron: 33 0 13,21,28,4 * *
+  steps:
+    cluster_profile: alibabacloud-qe
+    env:
+      E2E_RUN_TAGS: '@alibaba-ipi and @network-openshiftsdn'
+      TAG_VERSION: '@4.10'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-alibabacloud-ipi-fips
+- as: alibaba-ipi-fips-p3-f28-destructive
+  cron: 8 12 21 * *
```

Comment:
As the two are using the same workflow, how about another day except "13,21,28,4", e.g. 10?

Reply:
The cron was generated by https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh.